Makefile

```diff
@@ -164,14 +170,14 @@ ifeq ($(shell cd upstream && git rev-parse --is-shallow-repository), false)
 		git format-patch  -o ../upstream-patches --minimal --no-signature HEAD...$${LAST_TAG}
 endif
 
-patch_upstream: init_upstream export_upstream_patches
-	@# Ensure tool is installed
+upstream: init_upstream export_upstream_patches
```

Comment:
do we need both? since `export_upstream_patches` requires `init_upstream` already?